standards/webhooks.md

```diff
@@ -0,0 +1,48 @@
+# Webhooks
+
+## Overview
+
+Webhooks are automated messages sent from apps when something happens, typically used to integrate two disconnected systems together, such as with a third-party provider. They have a message—or payload—and are sent to a unique URL configured by the customer or user. Webhooks are an advantage in an API as they are almost always faster than polling for an updated status or parsing to determine which events or changes are new within a timeline. 
+
+- Services **SHOULD** implement push notifications, HTTP callbacks, and other event notifications via webhooks.
+- Services **MAY** implement HTTP callback endpoints as a webhook consumer from other systems and/or APIs.
+
+```note
+Push notification via HTTP Callbacks, often called Web Hooks, to publicly-addressable servers.
+```
+
+## Webhook Consumer
+
+Webhook consumption is necessary to integrate with other first-party and third-party systems. Often building specific API endpoints is necessary to receive webhook-specific payloads, headers and authorization in which the API producer cannot control or change. These webhook specific integration endpoints are usually not RESTful resources and may be required to deviate from expected design patterns within this API Style Guide. As a result webhook consumers should recognize this and organize webhook consuming endpoints accordingly.
+
+Webhook consuming endpoints:
+- **MAY NOT** follow API Standards or best practices when it does not pose inherit security risks or affect the rest of the API design consistency and experience.
+- **MUST** be marked as **internal** usage only in the design specification. Internal usage implies that the endpoint should only be used by the API Producer themselves, including the configuration of that endpoint by them against a third party system if needed. Internal usage does not imply the endpoint is not publicly addressable (i.e. in Open API Spec, this would translate to `x-internal: true`). <a name="sps-webhooks-internal" href="#sps-webhooks-internal"><i class="fa fa-check-circle" title="#sps-webhooks-internal"></i></a>
```

Comment:
"API Producer" you mean "API Owner/Developer" ? People might mix with "producer" as the API that initiates the call